docs/docs/events.md

```diff
@@ -16,7 +16,7 @@ Called each time this component appears on screen (attached to the view heirarch
 ```js
 class MyComponent extends Component {
   componentDidAppear() {
-    
+
```

Comment:
this is not even the Prettier, turns out that something else in Atom applies these changes 😓

Reply:
Damn, that's probably editorconfig

Reply:
Pardon me, I haven't touched Atom and JS for a few months. I revert most of changes, except these empty lines + I updated tests